atm.js

```diff
@@ -1,3 +1,106 @@
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+	var userAccounts = {
+		checkings : 0,
+		savings : 0
+	};
+
+	//deposit checking
+	$('#checking .deposit').on("click",function(){
+
+		moneyCheck()
+		inTheRedCheckings()
+	})
+	//deposit saving
+	$('#savings .deposit').on("click",function(){
+
+		moneySave()
+		inTheRedSavings()
+	})
```

Comment:
Any thoughts as to how you could go about creating a single event listener for deposit functionality, regardless of how many accounts there are?

Maybe you could pass in an argument to `moneySave` that indicates what account you're making a deposit into...